goals_vector:       [-4, -3, 
                     -4,  2,
                     -4,  7,
                      5, -7,
                      5, -3,
                      5,  1]

active_plan_algorithm:  "move_base"
                     
plan_algorithm: {move_base : 0, bug0 : 1}

wall_follow_active: false



